src/screens/MenuScreen.tsx

```diff
@@ -25,16 +26,10 @@ const randRecipeUrl = `https://api.spoonacular.com/recipes/random?apiKey=${API_K
 
 export default function MenuScreen() {
     // useDispatch allows us to dispatch Actions to mutate global store variables
-    const dispatch = useDispatch();
     const [randRecipes, setRandRecipes] = React.useState<Recipe[]>([initialState.recipe]);
-    const userState = useSelector<RootState, UserState>((state) => state.userState);
+    const [isCardStackLoading, setIsCardStackLoading] = React.useState<boolean>(false);
```

Comment:
I think the default should be ```true```,  actually. Even when we first come to the MenuScreen, we should wait for all data to load